includes/class-wc-payments-action-scheduler-service.php

```diff
@@ -116,15 +119,14 @@ private function track_order( $order_id, $is_update = false ) {
 	 * @param int    $timestamp - When the job will run.
 	 * @param string $hook      - The hook to trigger.
 	 * @param array  $args      - An array containing the arguments to be passed to the hook.
-	 * @param string $group     - The group to assign this job to.
 	 *
 	 * @return void
 	 */
-	public function schedule_job( $timestamp, $hook, $args = [], $group = '' ) {
```

Comment:
Would there be any circumstances where we'd want to have a custom group for a job? I'm just wondering whether it may be worth leaving the `group` parameter here but defaulting it to `self::GROUP_ID`? 

(I must admit that I can't think of a case where we'd want to do that off the top of my head, but I'm just thinking out loud!)

Reply:
I can't think of a reason why a different group would be needed, either. If we continue to assign the group in the parameter, then that will need to be done with all future methods, as well. It might be better to add the group back later on, if needed.

Reply:
Agreed! 👍 Thanks for addressing my comments.